web/src/admin/providers/oauth2/OAuth2ProviderFormForm.ts

```diff
@@ -178,18 +174,21 @@ export function renderForm(
                 >
                 </ak-hidden-text-input>
                 <ak-form-element-horizontal
+                    flow-direction="row"
```

Comment:
A bit of a loaded term in authentik. “flow direction” here refers to CSS Grid/Flex Box flow. 


https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction